### DISCUSSION TITLE
workshop: ESP-IDF a ESP32-C6 - %C3%9Akol 4 %C2%B7 Developer Portal

### DISCUSSION THREAD

**Discussion by giscus** [espressif/developer-portal#329] [General]
# workshop: ESP-IDF a ESP32-C6 - %C3%9Akol 4 %C2%B7 Developer Portal

The developer resources in just one place!

https://developer.espressif.com/cs/workshops/esp-idf-with-esp32-c6/assignment-4/

**Comment by XXXmathevXXX** [comment]
Bez zaškrtnuté možnosti "Partition Table -> Partition Table (Custom partition table CSV)" není možné vyvolat okno ESP-IDF Partition Table Editor.

**Comment by Honza0297** [comment]
Ahoj @XXXmathevXXX! Díky za nahlášení a omlouvám se za delší prodlevu. Otevřel jsem https://github.com/espressif/developer-portal/pull/341. kde mj. i zmiňuju nutnost nastavit danou možnost v konfiguraci. Ještě jednou díky!
